plugins/carbon_black_defense/help.md

```diff
@@ -392,3 +303,4 @@ The Find Event API and Get Details for Specific Event API are now documented as
 * [VMware Carbon Black Cloud Endpoint Standard API](https://developer.carbonblack.com/reference/carbon-black-cloud/cb-defense/latest/rest-api/)
 * [Rate Limits](https://developer.carbonblack.com/reference/carbon-black-cloud/rate-limiting/)
 * [API Key Types](https://developer.carbonblack.com/reference/carbon-black-cloud/authentication/#creating-an-api-key)
+* 
```

Comment:
Please update the Version History on line 290 with a major version bump and description of changes.

Reply:
Our current master branch version is 2.0.1 so this fix will be bundled with other updates (SOAR-8726 and SOAR-7132).

[SOAR-7132](https://github.com/rapid7/insightconnect-plugins/pull/1034)
[SOAR-8726](https://github.com/rapid7/insightconnect-plugins/pull/1163)